doc/site/index.html

```diff
@@ -0,0 +1,9 @@
+<html>
+  <head>
+    <script type="text/javascript">
+      // redirect to /latest version of docs
+      window.location.pathname = "latest"
+    </script>
+  </head>
```

Comment:
WIll this conflict with this: https://github.com/pachyderm/pachyderm/blob/master/netlify.toml#L6

Reply:
I don't know, but I now realize that it's at the very least unnecessary. I modified `serve.sh` to run `netlify dev`, which uses our `.netflify.toml` file instead of needing this.